tools/ci/ci_resources_unittest.sh

```diff
@@ -8,13 +8,12 @@ cd $WPT_ROOT
 main() {
     cd $WPT_ROOT
     pip install -U tox
-    pip install --requirement tools/wpt/requirements.txt
     ./wpt install firefox browser --destination $HOME
     ./wpt install firefox webdriver --destination $HOME/firefox
     export PATH=$HOME/firefox:$PATH
 
     cd $WPT_ROOT/resources/test
-    tox
+    tox -- --binary=$HOME/browsers/firefox/firefox
```

Comment:
The line above installs in `$HOME/firefox` - where does the `browsers/` subdir come from?

Reply:
This is installed in one more line above.

`wpt install firefox browser` installs the browser to `DESTINATION/browsers/firefox`